### PR TITLE
Fix wrong env var used

### PR DESCRIPTION
**Pull request by kalisp**
There is no `AYON_LOCAL_ID` environment variable, only `AYON_SITE_ID`

### CODE REVIEW

**Review by BigRoy — APPROVED** [review]
Seems correct. e.g. [here](https://github.com/ynput/ayon-core/blob/f3c9ffac1a5fde32cabe419d12025bb0c1e7165d/client/ayon_core/lib/local_settings.py#L572).

There's no other mention of `AYON_LOCAL_ID` or `AYON_SITE_ID` in the docs. So this seems the only place to fix it.

✅

**Review by MustafaJafar — APPROVED** [review]
Looks good.
I agree with BigRoy.

File: website/docs/addon_site_sync_admin.md [hunk 1]
```diff
@@ -200,7 +200,7 @@ As current implementation relies heavily on Settings and Site Settings, backgrou
 
 To do this:
 
-- run AYON `Tray` with environment variable AYON_LOCAL_ID set to name of active (source) site. In most use cases it would be studio (for cases of backups of everything published to studio site to different cloud site etc.)
+- run AYON `Tray` with environment variable AYON_SITE_ID set to name of active (source) site. In most use cases it would be studio (for cases of backups of everything published to studio site to different cloud site etc.)
 - start `Tray`
 - check `Local ID` in information dialog after clicking on version number in the Tray
```
Comment (MustafaJafar) [inline website/docs/addon_site_sync_admin.md:205]: Quick question: should this be site ID ?

Comment (BigRoy) [reply website/docs/addon_site_sync_admin.md:205]: Bonus points if we can add a screenshot showing the id in that dialog and maybe with arrows in the same image pointing to where to click in the tray menu, etc.

Comment (BigRoy) [reply website/docs/addon_site_sync_admin.md:205]: No bonus points @kalisp ? :)

**Pull request merged**